init_django.sh

```diff
@@ -27,5 +27,5 @@ if ! python manage.py shell -c \
 fi
 
 # Start webserver
-exec gunicorn -b [::]:8000 -w 1 django_evaluation.wsgi \
-    --log-level "$LOG_LEVEL" --error-logfile "$LOG_DIR/freva-web.error.log"
\ No newline at end of file
+exec gunicorn -b [::]:8000 --worker-class gevent -w 2 django_evaluation.wsgi \
```

Comment:
Setting this to -w 2 will break the plug-my-plugin button. This button is very annoying. We could drop it, but Etor will think otherwise.